### ISSUE TITLE
GCD for Wands going away with spell pushback

### ISSUE THREAD

**Issue by Truckez** [WeakAuras/WeakAuras2#1659]
**What steps will reproduce the problem?**
In Classic wow standard Weak Aura with Status>Global Cooldown. It works perfectly for me in retail. 

**What is the expected output? What do you see instead?**
Once I engage a mob it shows the GCD perfectly, but as soon as the mob starts attacking me (and I assume due to spell pushback), the Weak Aura disappears. I also tried to recreate the WA using just a progress bar.

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**
I turned off all others except WA and the problem persisted.

**Which version of WeakAuras are you using?**
2.14.6

**Was it working in a previous version? If yes, which was the last good one?**
It worked in retail, although I wasn't using a wand in retail.

**Do you have an error log of what happened?** 
No error log

**Please provide any additional information below:**

**If you do not know which aura is causing issues for you, please attach a ZIP archive of your WeakAuras SavedVariables file.** 

**In case you do know which aura is problematic, please export the string and paste it below.**

!nAvBZPnoq4Fn3hpILFbBFFJWlT0J4qrK2PZKreHTauJTLpj5qOFi)2VvYVah5ANMHbCKwV6z395zxrqK1emNGNxQzYD0u2JZtfLQhNN8Ln3XvPB(CntP5IY7OYNjyfb7oa5pyibxqWzMpW7xRyYnNijoeSM9Q2DPGxQj4XttwpDv72VA3MwRpiK3xz8OcEvWQt3VBNIPjjUXdcalktblUYdkAbBn45AjJCl4qzobFqRRu)1n3CKUxmGlU52tvB)7VfU4Z34breeeglTWzMW4Rzs(pE7PpxtZKun9TNwdaxXY3D1rLXuu4COAgzlblfAQbRKKya97(PoQ)CMLt3djP7Fy9I5jtj4Cb1MI4L7LI6kZJf15AUn212CZMs2lmP5000CgCcxBJQILAwubojNrsqN3hE(wsIh8L1Ym(UD8uyVtVZjsO2((ff5Sl8m(JthTWKe6mQBHw3d8dBY4A30GB33TEAovPU0)tw9W8jxzu1lv)S4M)J3bzDtUgicFvsRUtKbED8Ir3TemxtL6rnPiOyPkec9HLsXEjdqXwBUEu23RvAwgl7oEPTaZ2dr06tvGFQATT9ai4TaSYAoJrtMCrvgBqwIRpCk50sDJjZtWZNmThFxQPgLLzevFLrFEeWTupUuCKjTpEhlJtFSzvZ3Ha79hcrrNCryqzQOOQlkSGyAjfaxMnQ4GmCX4JoR(2QsWsdlqvLtpzeYai9cd8IGyHM(SHcwM1R3m(nxiBOrMpUWHoWlCOFOtquKRxGBSHCblg6h7GIh6eg57H8jj(MfJCS)HqXUoXE2coKCffgLkb)2NE7txkNUTha2iQ3YhQYmInCD7VNFJzcO60)gAjF)EMu1a319ly5xvS88eOjbSfkafc9r02skWj01GESUKBOAqwbEbSQEli5GYynixELG3mEeE9g86rRaFvcUPT1uMbggcpgKC7zLmjpfFqC8EyfL93rGG4f2zpUuYSEeVC6Ifg2fnh3Jnm2WibPWBpbTwBf)NH(TTXX8S2JVU06ZwAG1ABCag2UXhYfBP5V90yHiptCS0K92cbvdF(JtxSC2dl6s2BoWn0unVaiow3)lZGTnuuFVUSniP5qBxQ9Fw38MwIpunalbLsR2ZBOJPFhW9lP5FbQy8gEOP3oVG21bXkxnp0LL30wYyaVfsbTvWsrjZ4)ckV8336D8sU6WVT9MMnAE6ZNM0vXV1g7PsMMT58qahJKPjvBOUt4QEz4lDbQjoZRoqT9PpW47paAnuKx7WeBD2Gb(fZanAA0fJpsmW6Q(v0xnhZzzC38qSFGtmkOTXWyWjqMQWspUSJe6xTZ1DbqwzpQvNJqn6CFpNaeQhVxn4SGlLczFhQFTtTEjvkQSxByGpQHy1Kq6Pgxwi1gofx3nBO5AcZKMlgyRK4XRMonXmJURPX7qWSHHrOOyKRJxWW4HqlRD)zqKbvZcGEyErb(bUHEEoaCGD8oJ0(6ttJF42pV21f9prXTx7bIH)if6tKlo2LfqD9PV9Yk3faBgCMqB0OqhF)4yph7b7BbByCKBeKVD8DDqrWSMMDaWgHC9r(obWoHdrnHrBpzltMvyUpbgnWzq3nYaOvbjqysXhgpP5ImnHcc2ANjlcTtHBZa3ZHGpYZ0hijrMRJPfP9mBeYZX9Ym86)3jD3)FM01mFR1s1J4)PMkzB(6bUPxpV04CwF66NFtnHKde32UXFz6Q1ZhpArlf61okuN2SvITJVVJU8cGEFxtTK8Vp

**Comment by mrbuds** [comment]
Use `status > swing timer` with weapon = ranged for wands.
